src/styles/variables.json

```diff
@@ -3669,7 +3669,8 @@
         "1": "0.75rem",
         "2": "0.875rem",
         "3": "1rem",
-        "4": "1.25rem",
+        "4": "1.125rem",
+        "5": "1.25rem",
```

Comment:
We didn’t have a 5. Since 3 was 16px and 4 was 20px, and we needed 18px, I changed 4 to 18px and made 5 become 20px.